src/OmniSharp.MSBuild/ProjectLoadListener.cs

```diff
@@ -87,7 +87,7 @@ private static HashedString GetProjectId(ProjectLoadedEventArgs args)
 
         private static HashedString GetSdkVersion(ProjectLoadedEventArgs args)
         {
-            return _tfmAndFileHashingAlgorithm.HashInput(args.SdkVersion.ToString());
+            return _tfmAndFileHashingAlgorithm.HashInput($"{args.SdkVersion}");
```

Comment:
we could just return a `null` I think
however, a better question is if we should allow to start without `dotnet` on the path at all.
There would be other things that will fail without it e.g. auto restore or test adapter